R/f_simulation_multiarm_survival.R

```diff
@@ -546,7 +561,7 @@ getSimulationMultiArmSurvival <- function(design = NULL, ...,
 
     if (correlationComputation == "null") {
         # not accounting for alternative
-        corrMatrix <- matrix(rep(allocationRatioPlanned / (1 + allocationRatioPlanned), gMax^2), ncol = gMax, nrow = gMax)
+        corrMatrix <- matrix(rep(allocationRatioPlanned[1] / (1 + allocationRatioPlanned[1]), gMax^2), ncol = gMax, nrow = gMax)
```

Comment:
maybe a small comment here would be helpful to add why only the first element of `allocationRatioPlanned` is used could be helpful